keyboards/skyloong/dt40/readme.md

```diff
@@ -0,0 +1,34 @@
+# DT40
+
+![DT40](http://semitek.oss-cn-hangzhou.aliyuncs.com/qmk_imaging%2Fdt40%20layout05192023.jpg?OSSAccessKeyId=ogxQdo6geQXQVIK5&Expires=1684576837&Signature=GUNgvZ5GDL9uZY6CfpKYN1KvsJk%3D)
```

Comment:
please rehost on an image hosting site.

Reply:
This host is our own OSS, which provides a faster and more stable connection, and we want to continue using it.

Reply:
OK,Rehosted it.